lib/resources/audits.js

```diff
@@ -0,0 +1,17 @@
+// Copyright 2019 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/opendatakit/central-backend/blob/master/NOTICE.
+// This file is part of ODK Central. It is subject to the license terms in
+// the LICENSE file found in the top-level directory of this distribution and at
+// https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+// including this file, may be copied, modified, propagated, or distributed
+// except according to the terms contained in the LICENSE file.
+
+module.exports = (service, endpoint) => {
+
+  service.get('/audits', endpoint(({ Audit }, { auth, queryOptions }) =>
+    auth.canOrReject('audits.read', Audit.species())
```

Comment:
Just wanted to check, should this be `'audit.list'` rather than `'audits.read'`?

Reply:
yeah i went back and forth on that. ultimately since audits can't be individually addressed and retrieved over the API i decided that there was no need for both, and ultimately read is more descriptive than list.

Reply:
I think part of the reason I was surprised is that all the other roles use the singular case (for example, `user.list` and `user.read`). I think consistency around the case and the verb could be helpful, but I also don't have strong feelings if you have a preference for `audits.read`.

Reply:
oh oops you mean the plural. yeah i'll fix that then.